workbench/soapclient/SforceBaseClient.php

```diff
@@ -110,6 +111,19 @@ public function createConnection($wsdl, $proxy=null) {
             $soapClientArray['compression'] = SOAP_COMPRESSION_ACCEPT | SOAP_COMPRESSION_GZIP | 1;
         }
 
+        if (WorkbenchConfig::get()->value("ignoreHTTPSCertificateIssuesAllowed") &&
+                $connConfig->isIgnoreHTTPSCertificateIssuesEnabled()) {
+            $context = stream_context_create([
+                // set some relaxed SSL/TLS specific options for dev hosts
+                'ssl' => [
```

Comment:
this basically turns off any validation for HTTPS certs and quite dangerous thing unless user knows what they doing